packages/autocomplete-client/scripts/update_version.js

```diff
@@ -0,0 +1,12 @@
+const pkg = require('../package.json')
+const fs = require('fs')
+const path = require('path')
+
+const version = pkg.version
+const fileContents = `export const PACKAGE_VERSION = '${version}'
+`
+
+const filePath = path.resolve(__dirname, '../src/package-version.ts')
+
+console.log(filePath)
```

Comment:
If it's a debug log, maybe a sentence will be great. If not it should be remove

Reply:
I'm removing it